app/views/surveys/form_partials/_response_options_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<% response_options = survey.response_options.presence || Array.new(10, OpenStruct.new) %>
+
+<div id="nested-response-options">
+  <% response_options.each_with_index do |response_option, index| %>
+    <%= fields_for "survey[response_options][#{index}]", response_option do |fro|  %>
+      <div class="nested-response-option-form">
+        <div class="row">
+          <div class="col-12 col-sm-6">
+            <div class="form-group">
+              <label for="description"><%= index + 1 %>. Antwortmöglichkeit (deutsch)</label>
+              <%= fro.text_field :title_de, name: "survey[response_options][#{index}][title][de]", class: "form-control" %>
```

Comment:
max length validation of 150 chars is missing

Reply:
will be changed to html5 maxlength validation of text input with hard stop at 150 so users cannot type more

Reply:
required of first 2 is missing

Reply:
required validation for the first 2 is also missing.
this will be added per html5 as well, like:

![Bildschirmfoto 2021-06-21 um 11 33 48](https://user-images.githubusercontent.com/1942953/122740915-bccd0180-d284-11eb-82ed-e89ccf6e8681.png)

the downside is, that someone can fill in the third and fourth response option, which results in "min 2", but the form still requires the second one.